src/components/commons/Alram/index.tsx

```diff
@@ -0,0 +1,49 @@
+import Image from 'next/image';
+
+import { useEffect, useState } from 'react';
+
+import { useQuery } from '@tanstack/react-query';
+
+import { MyNotifications } from '@/apis/myNotifications';
+import { SVGS } from '@/constants';
+
+export const Alram = () => {
+  const [isExistedAlram, setIsExistedAlram] = useState<boolean>(false);
+  const [isActive, setIsActive] = useState<boolean>(false);
+
+  const { data, isSuccess } = useQuery({
+    queryKey: ['notifications'],
+    queryFn: () => MyNotifications.get(),
+    staleTime: 300 * 1000,
```

Comment:
const FIVE_MINUTES=1000*60*5
이렇게 사용하시면 어떨까요?

Reply:
```suggestion
    staleTime: TIMES.FIVE_MINUTES,
```
constants 폴더에 times.ts라는 파일을 만들어서 수정했습니다